bpf/Makefile.bpf

```diff
@@ -77,9 +77,15 @@ checkpatch:
 		-e GITHUB_REF=$(GITHUB_REF) -e GITHUB_REPOSITORY=$(GITHUB_REPOSITORY) \
 		$(CHECKPATCH_IMAGE_AND_ENTRY) $(CHECKPATCH_ARGS)
 
+# Note: --macro-file-builtins and --iso-file can be removed, here and in the
+# messages in the .cocci files, next time we upgrade coccinelle. The issue was
+# fixed in https://gitlab.inria.fr/coccinelle/coccinelle/-/commit/540888ff426e.
+COCCI_PATHS ?= --macro-file-builtins /usr/local/lib/coccinelle/standard.h --iso-file /usr/local/lib/coccinelle/standard.iso
```

Comment:
I haven't checked by setting `COCCINELLE_HOME` (couldn't find related documentation - although I didn't spend hours searching). I'll try that, if it works, yes it would be cleaner. Thanks!

Reply:
I'm just seeing your second comment - yeah, agree that the fix looks somewhat dubious. I found that it fixed the issue we had and didn't really look further to be honest.